Resources/Textures/Objects/Weapons/Grenades/smokegrenade.rsi/meta.json

```diff
@@ -0,0 +1,21 @@
+{
+  "version": 1,
+  "license": "CC-BY-SA-3.0",
+  "copyright": "",
```

Comment:
Why is it empty?

Reply:
What to write there?

Reply:
> What to write there?

Idk, `Created by <your GH nick> (GitHub)`

Reply:
forgot to mention, that if it actually was made by you